apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -103,7 +103,7 @@ type TempoStackSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tempo Component Templates"
 	Template TempoTemplateSpec `json:"template,omitempty"`
 
-	// ReplicationFactor is used to define how many component replicas should exist.
+	// The replication factor is a configuration setting that determines how many copies of your trace data are stored across different instances of Tempo's backend storage.
```

Comment:
cc @pavolloffay can you confirm?

Reply:
Thanks! @andreasgerstmayr 

> Here are docs for loki: https://grafana.com/docs/loki/latest/get-started/components/#replication-factor For Tempo I could only find the comment in the config, which reads: `# number of replicas of each span to make while pushing to the backend` (https://grafana.com/docs/tempo/latest/configuration/)

I tried to explain the thing based on the comment you pointed from the Tempo codebase. But it seems it is not ok.

What do you think about this?

```golang
// The replication factor is a configuration setting that determines many Ingesters need to acknowledge the data from the Distributors to avoid data loss.
```